pkg/injection/startup/files.go

```diff
@@ -120,22 +121,42 @@ func (runner *Runner) createCurlOptionsFile() error {
 }
 
 func (runner *Runner) createConfFile(path string, content string) error {
-	err := runner.fs.MkdirAll(filepath.Dir(path), onlyReadAllFileMode)
+	err := createConfigFile(runner.fs, path, content)
 	if err != nil {
 		return errors.WithStack(err)
 	}
 
-	file, err := runner.fs.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, onlyReadAllFileMode)
+	log.Info("created file", "filePath", path, "content", content)
+
+	return nil
+}
+
+func (runner *Runner) createConfFileWithShortMessage(path string, content string) error {
+	err := createConfigFile(runner.fs, path, content)
 	if err != nil {
 		return errors.WithStack(err)
 	}
 
-	_, err = file.Write([]byte(content))
+	log.Info("created file", "filePath", path)
+
+	return nil
+}
+
+func createConfigFile(fs afero.Fs, path string, content string) error {
```

Comment:
Having 3 `createConfigFile` functions that, 2 of them are just wrappers for nicer logs, is a bit hard to read (or it was for me at least 😅 )

could we make it nicer 🤔 
something like:
```go
func (runner *Runner) createConfigFile(path string, content string, verbose bool) error {
	err := createFile(runner.fs, path, content)
	if err != nil {
		return errors.WithStack(err)
	}
	if verbose {
		log.Info("created file", "filePath", path, "content", content)
	} else {
		log.Info("created file", "filePath", path)	
	}

	return nil
}
```
(renamed `createConfigFile` to `createFile`; combined the two `runner.createConfigFile...` to 1)

Reply:
`verbose` parameter added instead of many functions